docs/shortcuts.html

```diff
@@ -1,664 +1,837 @@
 {{+bindTo:partials.standard_devtools_article}}
 <h1>Keyboard Shortcuts</h1>
 
-    <p>
-      The DevTools has several built-in shortcut keys that developers can use to save time in their day to day workflow. Outlined below is each shortcut and the corresponding key for each on Windows/Linux and Mac. While some shortcuts are globally available across all of the DevTools, others are specific to a single panel, and are broken up based on where it can be used.
-    </p>
-
+<p>
+  The DevTools has several built-in shortcut keys that developers can use to save time in their day to day workflow. Outlined below is each shortcut and the corresponding key for each on Windows/Linux and Mac. While some shortcuts are globally available across all of the DevTools, others are specific to a single panel, and are broken up based on where it can be used.
+</p>
 
+<div class="collapsible">
+  <h2 id="opening-devtools"> Opening DevTools </h2>
+
+  <p> To access the DevTools, on any web page or app in Google Chrome you can use one of these options: </p>
+
+  <ul>
+    <li> Open the <b>Chrome menu </b> <img src="../images/chrome-menu.png" alt="Chrome menu"> at the top-right of your browser window, then select <b>Tools</b> &gt; <b>Developer Tools</b>.</li>
+    <li> Right-click on any page element and select <b>Inspect Element</b>.</li>
+  </ul>
+
+  <table>
+    <tbody>
+      <tr>
+        <th></th>
+        <th> Windows / Linux </th>
+        <th> Mac </th>
+      </tr>
+      <tr>
+        <td> Open Developer Tools </td>
+        <td> <span class="kbd">F12</span>, <span class="kbd">Ctrl</span> + <span class="kbd">Shift</span> + <span class="kbd">I</span> </td>
+        <td> <span class="kbd">Cmd</span> + <span class="kbd">Opt</span> + <span class="kbd">I</span> </td>
+      </tr>
+      <tr>
+        <td> Open / switch from inspect element mode and browser window </td>
+        <td> <span class="kbd">Ctrl</span> + <span class="kbd">Shift</span> + <span class="kbd">C</span> </td>
+        <td> <span class="kbd">Cmd</span> + <span class="kbd">Shift</span> + <span class="kbd">C</span> </td>
+      </tr>
+      <tr>
+        <td> Open Developer Tools and bring focus to the console </td>
+        <td> <span class="kbd">Ctrl</span> + <span class="kbd">Shift</span> + <span class="kbd">J</span> </td>
+        <td> <span class="kbd">Cmd</span> + <span class="kbd">Opt</span> + <span class="kbd">J</span> </td>
```

Comment:
In this table we use `Opt` in the next one we use `Alt` (it's the same key, right?).